make it clear which graph api is used

spottedmahn opened the pull request:
@saraford

azurecla commented:
Hi __@LuisCabrer__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (luisca). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

spottedmahn committed:
make it clear which graph api is used

parakhj commented on articles/active-directory-b2c/active-directory-b2c-devquickstarts-graph-dotnet.md:
Thanks for writing this up! Could you update lines 22 to line 25 to the following (include the > characters)

>[!NOTE]
>Currently you must use the [Azure AD Graph API](https://msdn.microsoft.com/Library/Azure/Ad/Graph/howto/azure-ad-graph-api-operations-overview?f=255&MSPPError=-2147217396) to manage the users in an Azure AD B2C directory. In the future, [Microsoft Graph](https://developer.microsoft.com/en-us/graph) will become the API. Learn more [here](https://dev.office.com/blogs/microsoft-graph-or-azure-ad-graph).

spottedmahn replied:
@parakhj done 😄

spottedmahn committed:
updated based upon Parakh's feedback

spottedmahn committed:
remove graph api in the future

per the rules mentioned by @parakhj

> he internal doc rules prevents us from stating what's coming in the future
